_WINDOWS_ is pre-defined macro definition for Windows OS
Other libraries detect build on Windows by this macro
Fix SLADE with wxWidgets 3.2
Index: FreeImage/Source/FreeImage.h
===================================================================
--- FreeImage.orig/Source/FreeImage.h
+++ FreeImage/Source/FreeImage.h
@@ -129,7 +129,6 @@ FI_STRUCT (FIMULTIBITMAP) { void *data;
 #endif // __MINGW32__
 
 #ifndef _WINDOWS_
-#define _WINDOWS_
 
 #ifndef FALSE
 #define FALSE 0
